Userland/Utilities/hostname.cpp

```diff
@@ -6,6 +6,7 @@
 
 #include <LibCore/ArgsParser.h>
 #include <errno.h>
+#include <limits.h>
```

Comment:
Our `HOST_NAME_MAX` is not defined in limits.h, is it supposed to be?

Reply:
SerenityOS currently defines it inside `unistd.h`, POSIX mandates `limits.h`. I'll amend the PR to move it to its proper place.